python/cuml/test/test_fil.py

```diff
@@ -406,19 +406,18 @@ def test_lightgbm(tmp_path):
     num_round = 5
     bst = lgb.train(param, train_data, num_round)
     gbm_preds = bst.predict(X)
-
     model_path = str(os.path.join(tmp_path,
                                   'lgb.model'))
     bst.save_model(model_path)
     fm = ForestInference.load(model_path,
                               algo='TREE_REORG',
-                              output_class=False,
+                              output_class=True,
                               model_type="lightgbm")
 
     fil_preds = np.asarray(fm.predict(X))
     fil_preds = np.reshape(fil_preds, np.shape(gbm_preds))
 
-    assert np.allclose(gbm_preds, fil_preds, 1e-3)
+    assert np.array_equal(np.round(gbm_preds), fil_preds, 1e-3)
```

Comment:
The 3rd parameter in `np.array_equal` is a boolean indicating whether to compare NaNs as equal (https://numpy.org/doc/stable/reference/generated/numpy.array_equal.html). Since this is probably not what is intended, it's best just to remove it.